File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_TRINHDOCHUYENMON_Service.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_TRINHDOCHUYENMON_Service : BaseService<NS_DM_TRINHDOCHUYENMON_, INS_DM_TRINHDOCHUYENMON_Repository>
    , INS_DM_TRINHDOCHUYENMON_Service
    {
    private readonly INS_DM_TRINHDOCHUYENMON_Repository _NS_DM_TRINHDOCHUYENMON_Repository;
    public NS_DM_TRINHDOCHUYENMON_Service(INS_DM_TRINHDOCHUYENMON_Repository NS_DM_TRINHDOCHUYENMON_Repository) : base(NS_DM_TRINHDOCHUYENMON_Repository)
    {
    _NS_DM_TRINHDOCHUYENMON_Repository = NS_DM_TRINHDOCHUYENMON_Repository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_TRINHDOCHUYENMON_.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_TRINHDOCHUYENMON_ } from './NS_DM_TRINHDOCHUYENMON_.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_TRINHDOCHUYENMON_Service extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaTDChuyenMon', 'TenTDChuyenMon'];

    List: NS_DM_TRINHDOCHUYENMON_[] | undefined;
    ListFilter: NS_DM_TRINHDOCHUYENMON_[] | undefined;
    FormData!: NS_DM_TRINHDOCHUYENMON_;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_TRINHDOCHUYENMON_";
    }
}

